lib/autoupdate/agent/process.go

```diff
@@ -371,15 +372,18 @@ func (s SystemdService) Enable(ctx context.Context, now bool) error {
 	if err := s.checkSystem(ctx); err != nil {
 		return trace.Wrap(err)
 	}
-	args := []string{"enable", s.ServiceName}
-	if now {
-		args = append(args, "--now")
-	}
-	code := s.systemctl(ctx, slog.LevelInfo, args...)
+	// The --now flag is not supported in systemd versions older than 219.
+	code := s.systemctl(ctx, slog.LevelInfo, "enable", s.ServiceName)
```

Comment:
```suggestion
	// The --now flag is not supported in systemd versions older than 219,
	// so perform enable + start commands instead.
	code := s.systemctl(ctx, slog.LevelInfo, "enable", s.ServiceName)
```

As written, the comment makes sense when you can see the diff, but I think it might get confusing without additional context.